examples/simple/main.tf

```diff
@@ -18,6 +18,13 @@ provider "google" {
   region = "us-central1"
 }
 
+resource "google_pubsub_schema" "example" {
```

Comment:
Should we actually just move this into the module? So an input variable could be:

```
schema = {
  name = "example"
  type = "AVRO"
  definition = "{\n \"type\" : \"record\",\n \"name\" : \"Avro\",\n \"fields\" : [\n {\n \"name\" : \"StringField\",\n \"type\" : \"string\"\n },\n {\n \"name\" : \"IntField\",\n \"type\" : \"int\"\n }\n ]\n}\n"
  encoding = "JSON"
}
```

Just a thought.